articles/design/using-auth0-with-multi-tenant-apps.md

```diff
@@ -9,50 +9,48 @@ contentType: concept
+You can call `/authorize` with a specific connection for a user, using the `connection` option in the [Auth0 SPA SDK](/libraries/auth0-spa-js), or by passing a `connection` parameter to the `authorize()` method in [Auth0.js](/libraries/auth0js/v9).
 
-* Use multiple [connections](/connections).
-* Identify different tenants by application (using client IDs).
-* Store tenant details in [app_metadata](/users/concepts/overview-user-metadata).
+### Identify tenants by application
 
-Another approach would be to use separate Auth0 tenants for each of your customers.
+In this scenario, each of your tenants would be a separate application in Auth0. This allows you to configure each slightly differently and to enable or disable connections per application at will, if your customers/tenants have different needs. This will require you to keep track of which tenant each of your users belongs to in your application, and then when logging them in, specify a connection to let them use. 
 
-If you're planning to build a multi-tenant solution, please [contact our Sales or Professional Services](https://auth0.com/?contact=true) teams.
+This method would require you to use a different set of Auth0 credentials for the users belonging to each customer, as each would be a different application.
 
-::: note
-We recommend that you [create multiple Auth0 tenants](https://github.com/auth0/auth0-multitenant-spa-api-sample) only if you need to share access to the Auth0 Dashboard with individual customers.
-:::
+### Store tenant details in app_metadata
 
-### Use multiple connections
+This is perhaps the simplest and most straightforward scenario, in which you simply add an `app_metadata` field for the tenant to [your metadata](/users/concepts/overview-user-metadata#metadata-usage), whatever identifier makes sense to you (such as `"tenant": "customer_12345"`). This method allows all of your users, despite what customer/tenant they belong to, to log in in one place with one user pool. You'll be able to check for this value in your application to sort users into the correct place after they login and are returned to your application.
 
-::: warning
-[Lock](https://auth0.com/lock) supports a maximum of **50 Database Connections** per [application](/applications). Enterprise Connections are not affected by this limit.
-:::
+### Create separate Auth0 tenants for each customer
 
-While using multiple [Connections](/identityproviders) introduces additional layers of complexity, there are several scenarios where this option might make sense:
+We recommend that you follow this approach only if you need to share access to the Auth0 Dashboard with individual customers. Otherwise, one of the above solutions is a more practical and easy to manage one than attempting to manage many Auth0 tenant dashboards, which is also not a scalable solution as your customer base grows.

```

Comment:
You can create a separate Auth0 tenant for each of your tenants.

We recommend that you follow this approach only if you need to share access to the Auth0 Dashboard with individual customers. Otherwise, one of the above solutions is a more practical and easy to manage one than attempting to manage many Auth0 tenant dashboards, which is also not a scalable solution as your customer base grows.